docs/skillmap/collector/collector5.md

```diff
@@ -495,6 +495,5 @@ Need help thinking of something to do?
 
 
 
-When you're done, click **Finish** to publish your project
-and share with friends and family!
-
+Click **Finish** to return to the main page where you can share your game
```

Comment:
There's a lot of sharing "with family and friends". Maybe, occasionally, just "share" or "share with others". My family and friends don't care. 😦

Reply:
WE WANT YOU TO MAKE THEM CARE!!!!! lol